src/pages/Home_styles.js

```diff
@@ -0,0 +1,57 @@
+import { Link } from 'react-router-dom';
+import Styled from 'styled-components';
+
+const StyledHeadline = Styled.h1`
+  color: ${({ theme }) => theme.colors.black};
+  text-align: center;
+  margin: 30px auto 10px;
+`;
+
+const StyledInfo = Styled.section`
+  width: 740px;
+  margin: 55px auto 20px;
+  padding-right: 0p;
+  text-align: left;
+
+  &:last-child {
+    margin-bottom: 100px;
+  }
+`;
+
+const StyledLink = Styled(Link)`
+  color: #0087ff;
```

Comment:
This should be the default color of a link (set in the global styles). It should be in the theme as well.